cfbs/prompts.py

```diff
@@ -34,7 +34,7 @@ def prompt_user(non_interactive, prompt, choices=None, default=None):
         if answer == "":
             answer = default
         elif choices and answer not in choices and answer.lower() not in choices:
-            print("Invalid value entered, must ve one of: %s" % "/".join(choices))
+            print("Invalid value entered, must be one of: %s" % "/".join(choices))
```

Comment:
This should be on a separate commit, please see https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#commits